src/Post.js

```diff
@@ -1,31 +1,40 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import "./index.css";
 import Avatar from "@material-ui/core/Avatar";
 import FavoriteBorder from "@material-ui/icons/FavoriteBorder";
 import ChatBubbleOutlineOutlinedIcon from "@material-ui/icons/ChatBubbleOutlineOutlined";
-import data from './data/database.json';
+// import data from './data/database.json';
+import { database } from './firebase';
 
 const Post = () => {
   const [likeCounter, setLikeCounter] = useState(0);
+  const[posts, setPosts] = useState([]);
 
   // Increase the total likes of the post after 'like' is clicked
   // const handleLikeClick = (id) => {
   //   const likedPost = data.posts.filter( post => post.id === id );
   //   console.log(likedPost.map( post => (post.likes + 1) ));
   // }
 
-  const handleLikeClick = () => {
+  const handleLikeClick = (like) => {
     setLikeCounter( (likeCounter) => {
       return likeCounter + 1;
     });
   }
 
-
+  useEffect(() => {
+    database.collection('posts').orderBy("timestamp", "desc").onSnapshot(snapshot => {
+      setPosts(snapshot.docs.map( doc => ({
+        id: doc.id,
+        post: doc.data()
+      }) ));
+    })
+  }, []);
```

Comment:
Pair programming: Anonna, Quang and Corinne
To link Post.js to the database (Firebase)